lib/OpenQA/WebSockets/Server.pm

```diff
@@ -279,6 +284,11 @@ sub setup {
     # start worker checker - check workers each 2 minutes
     Mojo::IOLoop->recurring(120 => \&_workers_checker);
 
+    my $service = OpenQA::IPC->ipc->{bus}->get_service("org.opensuse.openqa.Scheduler");
+    my $scheduler = $service->get_object("/Scheduler", "org.opensuse.openqa.Scheduler");
+
+    $scheduler->connect_to_signal("JobsAvailable" => \&jobs_available);
```

Comment:
Actually it is. It is using dbus type notification so systemd considers the service started up only when specified service appear on dbus. And only then the websockets and webui is started.